fe1-web/src/core/network/jsonrpc/messages/Message.ts

```diff
@@ -132,26 +130,31 @@ export class Message {
     this.message_id = msg.message_id;
     this.witness_signatures = [...msg.witness_signatures];
 
-    if (msg.channel) {
-      this.channel = msg.channel;
-    }
-
     const jsonData = msg.data.decode();
     const dataObj = JSON.parse(jsonData);
-    this.#messageData = messageRegistry.buildMessageData(dataObj);
+
+    let laoId: Hash | undefined;
+    try {
+      laoId = getLaoIdFromChannel(channel);
+    } catch {
+      laoId = undefined;
```

Comment:
I am not so sure, there will be some false positives because it would then warn every time for `lao#create` messages, right? (They are sent on the root channel)

Reply:
Yes exactly :) At the moment this is only the case for the first message to create the lao, before there is no lao channel so yeah 😅